internal/core/graphics/image.rs

```diff
@@ -991,47 +991,21 @@ impl BorrowedOpenGLTextureBuilder {
     }
 }
 
-#[cfg(feature = "unstable-wgpu-24")]
-#[derive(Debug)]
-#[non_exhaustive]
-/// This enum describes the possible errors that can occur when importing a WGPU texture,
-/// via [`Image::try_from()`].
-pub enum WGPUTextureImportError {
-    /// The texture format is not supported. The only supported format is Rgba8Unorm and Rgba8UnormSrgb.
-    InvalidFormat,
-    /// The texture usage must include TEXTURE_BINDING as well as RENDER_ATTACHMENT.
-    InvalidUsage,
-}
-
-#[cfg(feature = "unstable-wgpu-24")]
-impl core::fmt::Display for WGPUTextureImportError {
-    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
-        match self {
-            WGPUTextureImportError::InvalidFormat => f.write_str(
-                "The texture format is not supported. The only supported format is Rgba8Unorm and Rgba8UnormSrgb",
-            ),
-            WGPUTextureImportError::InvalidUsage => f.write_str(
-                "The texture usage must include TEXTURE_BINDING as well as RENDER_ATTACHMENT",
-            ),
-        }
-    }
-}
-
 #[cfg(feature = "unstable-wgpu-24")]
 impl TryFrom<wgpu_24::Texture> for Image {
```

Comment:
IMHO this implementation should also go in the wgpu_24 module

Reply:
Excellent point. I'll do that.